clients/drcachesim/tools/filter/type_filter.h

```diff
@@ -1,5 +1,5 @@
 /* **********************************************************
```

Comment:
> (because the flag constant being checked is a single bit).

I think with TESTANY the order doesn't matter, no matter the bit count. For TESTALL, incorrect param order would've been a correctness bug.

Reply:
Agreed.  Updated the description.